docs/source/tools/assistant.rst

```diff
@@ -7,7 +7,7 @@ Daml Assistant (``daml``)
 ``daml`` is a command-line tool that does a lot of useful things related to the SDK. Using ``daml``, you can:
 
 - Create new Daml projects: ``daml new <path to create project in>``
-- Create a new project based on `create-daml-app <https://github.com/digital-asset/create-daml-app>`_: ``daml create-daml-app <path to create project in>``
+- Create a new project based on the create-daml-app_ template: ``daml new --template=create-daml-app <path to create project in>``
```

Comment:
is that link actually useful? why do we want to link to our repo here?

Reply:
There used to be a link, so I thought I'd preserve that. The main advantage I can think of is that it lets people see what the project would look like without creating it. Agree that's minimal, happy to remove it.

Reply:
With all the `.template` files in there, I’d say it’s probably more confusing than helpful so I’d just remove it.